packages/kubernetes-1.15/kubelet-config

```diff
@@ -29,6 +29,24 @@ authorization:
 clusterDomain: {{settings.kubernetes.cluster-domain}}
 clusterDNS:
 - {{settings.kubernetes.cluster-dns-ip}}
+{{~#if settings.kubernetes.eviction-hard}}
+evictionHard:
+  {{join_map ": " "\n  " "no-fail-if-missing" settings.kubernetes.eviction-hard}}
+{{~/if}}
+kubeReserved:
+  {{~#each settings.kubernetes.kube-reserved}}
+  {{~#if (eq @key "cpu")}}
+  {{@key}}: "{{this}}"
+  {{~else}}
+  {{@key}}: {{this}}
+  {{~/if}}
+  {{~/each}}
+{{~#if settings.kubernetes.cpu-manager-policy}}
+cpuManagerPolicy: {{settings.kubernetes.cpu-manager-policy}}
```

Comment:
cpuManagerPolicy setting has default value of `None` should we leverage that here ? If this setting is not set in kubeconfig file, Kubernetes would assume as `None`, so this would work too. But should we explicitly set None and avoid conditional setting ? I would leave it for others to recommend best way forward here.

Reply:
same as above.😃